git-hooks/git_hooks/pre-commit.d/author

```diff
@@ -21,6 +22,6 @@ then
     if [[ ! $REPLY =~ ^[Yy]$ ]]
     then
         echo "You can change the user by running \"git su <name>\""
-        return 1
+        exit 3
```

Comment:
Any reason why taking `3`

Reply:
Yup! Unique letters are easier to debug since the exit code is usually mentioned. So all is better than 1 I guess.